microprofile/opentracing-extension/src/main/java/org/wildfly/extension/microprofile/opentracing/SubsystemDefinition.java

```diff
@@ -43,4 +58,14 @@ protected SubsystemDefinition() {
     public Collection<AttributeDefinition> getAttributes() {
         return Collections.emptyList();
     }
+
+    @Override
+    public void registerAdditionalRuntimePackages(final ManagementResourceRegistration resourceRegistration) {
```

Comment:
The same thing should be done for other MicroProfile subsystem (metrics, config, health).
@jfdenise Were you planning to do those in this PR or should I open another one?

Reply:
I did it at some point, but this PR doesn't reflect it. Adding them in the same PR.